### DISCUSSION TITLE
Add an option to have localized moment instances

### DISCUSSION THREAD

**Discussion by umaknow-lp** [microsoft-search/pnp-modern-search#1605] [Ideas]
**Is your feature request related to a problem? Please describe.**
In the handleBar helpers, when the moment instance in created, it does not have the localized version of moment, for performance purpose as stated [in the gulpfile at line 79](https://github.com/microsoft-search/pnp-modern-search/blob/main/search-parts/gulpfile.js), which make it impossible to format dates in another language.

**Describe the solution you'd like**
I would like to either have an option to activate localization for the handlebar helpers, or create a different version of the .sppkg that will have the localization activated if the first option is not possible.

**Comment by wobba** [comment]
Localization will use the current page's language or the language of a translated page. We don't plan to add other options. The page language should be sufficient (which is based on the browser or profile language if the language is enabled in the SP site).
